carma/test/transform_server_integration.test

```diff
@@ -22,7 +22,6 @@
   Also sets up all static transforms used by tf2 within the system.
 
   Use:
-  First delete the generated build/scripts folder in all java packages.
```

Comment:
You can actually delete this whole file and the carma_transform_server package. It was only written to support Java nodes and isn't needed for c++